ui/core/components/suggest_reforges_action.tsx

```diff
@@ -625,14 +628,105 @@ export class ReforgeOptimizer {
 		);
 	}
 
+	buildSoftCapBreakpointsLimiter({ useSoftCapBreakpointsInput }: { useSoftCapBreakpointsInput: BooleanPicker<Player<any>> | null }) {
+		if (!useSoftCapBreakpointsInput) return null;
```

Comment:
Does this also need a check for `this.enableBreakpointLimits`?

Reply:
Yep, you're right. Added!